internal/cli/atlas/networking/containers/delete.go

```diff
@@ -53,11 +53,11 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <containerId>",
 		Aliases: []string{"rm"},
-		Short:   "Remove the specified network peering container from your project.",
+		Short:   "Remove the specified network peering container from your project before creating any clusters. Don't run this command if you have clusters in your project.",
 		Long:    fmt.Sprintf(usage.RequiredRole, "Project Owner"),
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"containerIdDesc": "Unique 24-hexadecimal digit string that identifies the network container that you want to remove.",
+			"containerIdDesc": "Unique 24-hexadecimal digit string that identifies the network container that you want to remove",
```

Comment:
@kanchana-mongodb I think what @kyuan-mongodb means is you deleted this period, was that intentional? could we rollback this change?
```suggestion
			"containerIdDesc": "Unique 24-hexadecimal digit string that identifies the network container that you want to remove.",
```

Reply:
Thanks @gssbzn!